contract/AElf.Contracts.Genesis/BasicContractZero_Helper.cs

```diff
@@ -387,4 +460,10 @@ public static Address BuildContractAddress(int chainId, long serialNumber)
     {
         return BuildContractAddress(HashHelper.ComputeFrom(chainId), serialNumber);
     }
+
+    public static Address BuildContractAddressWithSalt(Address deployer, Hash salt)
```

Comment:
Can we change it to `ComputeContractAddress` to align with the methods it references?